src/harfbuzz-config.cmake

```diff
@@ -0,0 +1,5 @@
+get_filename_component(_harfbuzz_rootdir ${CMAKE_CURRENT_LIST_DIR}/../../../ ABSOLUTE)
```

Comment:
If `libdir` is multiple components (e.g., Debian), this is not a correct computation.